src/components/card/CardSidebarTabDetails.vue

```diff
@@ -316,6 +334,10 @@ export default {
 </script>
 <style lang="scss" scoped>
 
+.section-wrapper::v-deep .mx-datepicker-main .mx-datepicker-popup {
```

Comment:
```suggestion
.section-wrapper::v-deep .mx-datepicker-main.mx-datepicker-popup {
```

Both classes are applied on the same element

![image](https://user-images.githubusercontent.com/3404133/103888715-76745380-50e5-11eb-9c22-d8e345db39bb.png)